File: php_tasks/task6.php
<?php
    $a=readline ("Enter a:");
    $b=readline ("Enter b:");
    $c = $a;
    $sum=0;
    while ($c >= $a && $c <= $b)
    {
        $sum += $c;
        $c++;
    }
    echo "Sum: ", $sum;

File: cpp_tasks/task6.cpp
#include <iostream>
using namespace std;
int main()
{
    int a, b, c, sum;
    cout << "Enter a, b: " << endl;
    cin >> a >> b;
    c = a;
    sum=0;
    while (c >= a && c <= b)
    {
        sum += c;
        c++;
    }
    cout << "Sum: " << sum;
}